api/src/middleware/assessmentsDummyRouter.ts

```diff
@@ -0,0 +1,291 @@
+import { Router } from 'express';
+
+import { itemEnvelope } from './responseEnvelope';
+import { BadRequestError } from './httpErrors';
+import {
+  insertToProgramParticipants,
+  insertToAssessmentSubmissions,
+} from '../services/assessmentsDummyService';
+const assessmentsDummyRouter = Router();
+
+interface ProgramParticipantsRow {
+  id: number;
+  principal_id?: number;
+  program_id?: number;
+  role_id?: number;
+}
+
+assessmentsDummyRouter.get(
+  '/makeParticipant/:programId/:participantId',
+  async (req, res, next) => {
+    const { programId, participantId } = req.params;
+    // a role ID of 1 corresponds to a participant
+    const roleId = 1;
+
+    const programIdParsed = Number(programId);
+    const participantIdParsed = Number(participantId);
+    if (
+      !Number.isInteger(programIdParsed) ||
+      programIdParsed < 1 ||
+      !Number.isInteger(participantIdParsed) ||
+      participantIdParsed < 1
+    ) {
```

Comment:
These cannot be combined. If you found this programming paradigm in another file and have adopted it for your own, and if they were separate in that other file, there may be a good reason. Please separate out these tests and separate out their calls to `next()`.